version.mak

```diff
@@ -6,7 +6,7 @@
 # increment after releasing/branching
 VERSION_MAJOR=4
 VERSION_MINOR=5
-VERSION_PATCH_LEVEL=7
+VERSION_PATCH_LEVEL=8
```

Comment:
PR says post 4.5.6 but this one looks like a post 4.5.7

Reply:
Fixed, sorry